ScreenToGif/Resources/Localization/StringResources.it.xaml

```diff
@@ -111,11 +122,26 @@
     <s:String x:Key="Interface.EditorBackground.GridSize.Apply">Applica</s:String>
 
     <s:String x:Key="Interface.Other">Altro</s:String>
-    <s:String x:Key="Interface.Other.ExtendChrome">Estendi la barra del titolo/Chrome.</s:String>
+    <s:String x:Key="Interface.Other.ExtendChrome">Estendi la barra del titolo (finestra chrome).</s:String>
```

Comment:
Hi, "chrome" here means "window chrome" also known as "window titlebar", is it the same word/meaning in Italian?